code_conventions.md

```diff
@@ -112,21 +79,19 @@ else // else on new line
 { // A code block is always possible in this case, so use it
     cout << "Hahaha" << EOL;
 }
-~~~~~~~~~~~~~~~
-
-Naming conventions
-------
- * variables: lower_case_with_underscores
- * functions: lowerCamelCase
- * classes: UpperCamelCase
- * macros/constants: UPPER_CASE_WITH_UNDERSCORES
+```
 
-Function names should start with a verb (e.g. get, set, run, execute, validate etc.) or a question (e.g. is, has, can) as this helps a lot with understanding what the implementation is about.
+## Naming conventions
+ * **variables**: lower_case_with_underscores
+ * **functions**: lowerCamelCase
+ * **classes**: UpperCamelCase
+ * **macros/constants**: UPPER_CASE_WITH_UNDERSCORES
```

Comment:
I would argue to treat macros and constants differently, using all upper case for macros is ok to clearly indicate it is a different construct, but I would use upper camel case for constants, since that leads to nicer to read code.

Reply:
This is what we have been doing, if you want to discuss this, feel free to do so at the architecture meeting.